proof/refine/X64/Interrupt_R.thy

```diff
@@ -500,6 +500,13 @@ crunch X64_H.updateIRQState
   and ex_cte_cap_wp_to'[wp]: "ex_cte_cap_wp_to' a b"
   (simp: ex_cte_cap_wp_to'_def valid_mdb'_def)
 
+crunch set_irq_state
+  for valid_arch_state[wp]: valid_arch_state
+
+lemma maxUserIRQ_le_maxIRQ:
+  "X64.maxUserIRQ \<le> maxIRQ"
+  by (simp add: X64.maxUserIRQ_def maxIRQ_def)
```

Comment:
worth moving this somewhere more interface-related in CRefine?

Reply:
maxUserIRQ is an X64 concept, I don't think the other architectures have that